Snowflake.Data.Tests/SFDbTransactionIT.cs

```diff
@@ -118,7 +118,7 @@ public void TestTransactionRollbackOn2Transactions()
             conn.Open();
 
             DbCommand command = conn.CreateCommand();
-            command.CommandText = "create or replace table testTransaction(x timestamp, a integer)";
+            command.CommandText = "create or replace table testTransaction(x timestamp_ntz, a integer)";
```

Comment:
why change the data type?

Reply:
Same as above. Maybe such explanation should be given somewhere in jira? Or github issue comment?